src/TrueOrFalse.Frontend.Web/Views/ApiVueApp/Components/User/GoogleController.cs

```diff
@@ -64,32 +66,6 @@ public bool UserExists(string googleId)
         return _userReadingRepo.GoogleUserExists(googleId);
     }
 
-    public readonly record struct CreateAndLoginResult(
-        bool Success,
-        VueSessionUser.CurrentUserData Data,
-        string MessageKey);
-
-    [HttpPost]
-    public CreateAndLoginResult CreateAndLogin(GoogleUserCreateParameter googleUser)
```

Comment:
Please check the register flow and make sure, that this action is not in use anymore.